### PR TITLE
hindi-trans

### PR DESCRIPTION
**Pull request by willtryagain**
Update Appwrite's Hindi language translations. 

## Issues

[Issue-1690](https://github.com/appwrite/appwrite/issues/1690)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
Yes

### CODE REVIEW

**Review by rvkantpujari — CHANGES_REQUESTED** [review]
Some translations are not correct. Looks like you just translated from Google Translator. But still, most of that is 👍🏻

**Comment by willtryagain** [comment]
@rvkantpujari could you please specify the mistakes. I can't view your suggested changes.

File: app/config/locale/translations/hi.json [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "खाता सत्यापन",
+    "emails.verification.hello": "अरे {{name}}",
```
Comment (rvkantpujari) [inline app/config/locale/translations/hi.json:7]: नमस्ते {{name}}

File: app/config/locale/translations/hi.json [hunk 2]
```diff
@@ -3,30 +3,30 @@
...
+    "emails.verification.body": "अपना ईमेल पता सत्यापित करने के लिए इस लिंक का अनुसरण करें।",
+    "emails.verification.footer": "अगर आपने इस पते को सत्यापित करने के लिए नहीं कहा है, तो आप इस संदेश को अनदेखा कर सकते हैं।",
```
Comment (rvkantpujari) [inline app/config/locale/translations/hi.json:9]: यदि आपने इस पते को सत्यापित करने के लिए नहीं कहा है, तो आप इस संदेश को अनदेखा कर सकते हैं।

File: app/config/locale/translations/hi.json [hunk 3]
```diff
@@ -3,30 +3,30 @@
...
+    "emails.verification.thanks": "धन्यवाद",
+    "emails.verification.signature": "{{project}} टीम",
+    "emails.magicSession.subject": "लॉग इन",
+    "emails.magicSession.hello": "अरे,",
```
Comment (rvkantpujari) [inline app/config/locale/translations/hi.json:13]: नमस्ते

File: app/config/locale/translations/hi.json [hunk 4]
```diff
@@ -3,30 +3,30 @@
...
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "खाता सत्यापन",
+    "emails.verification.hello": "अरे {{name}}",
+    "emails.verification.body": "अपना ईमेल पता सत्यापित करने के लिए इस लिंक का अनुसरण करें।",
```
Comment (rvkantpujari) [inline app/config/locale/translations/hi.json:8]: अपना ईमेल सत्यापित करने के लिए इस लिंक का अनुसरण करें।

**Review by rvkantpujari — COMMENTED** [review]
Here are some changes. 😇👍🏻

File: app/config/locale/translations/hi.json [hunk 5]
```diff
@@ -3,30 +3,30 @@
...
+    "emails.recovery.footer": "यदि आपने अपना पासवर्ड रीसेट करने के लिए नहीं कहा है, तो आप इस संदेश को अनदेखा कर सकते हैं।",
+    "emails.recovery.thanks": "धन्यवाद",
+    "emails.recovery.signature": "{{project}} टीम",
+    "emails.invitation.subject": "%s टीम को %s पर आमंत्रण",
+    "emails.invitation.hello": "नमस्ते",
+    "emails.invitation.body": "यह मेल आपको इसलिए भेजा गया था क्योंकि {{owner}} आपको {{project}} पर {{team}} टीम का सदस्य बनने के लिए आमंत्रित करना चाहता था।",
+    "emails.invitation.footer": "यदि आप रुचि नहीं रखते हैं, तो आप इस संदेश को अनदेखा कर सकते हैं।",
+    "emails.invitation.thanks": "धन्यवाद",
+    "emails.invitation.signature": "{{project}} टीम",
     "locale.country.unknown": "अज्ञात",
```
Comment (christyjacob4) [inline app/config/locale/translations/hi.json:30]: A lot of these translations have been picked up from Google Translate and seem to lack a natural flow. Please fix them

**Review by christyjacob4 — CHANGES_REQUESTED** [review]

File: app/config/locale/translations/hi.json (hunk 5 above)
Comment (christyjacob4) [reply app/config/locale/translations/hi.json:30]: We do not prefer translations from Google Translate

Comment (harshita214) [reply app/config/locale/translations/hi.json:30]: Can I fix this?

Comment (willtryagain) [reply app/config/locale/translations/hi.json:30]: I have tried to be more original this time. Please have a look.

**Review by christyjacob4 — APPROVED** [review]
Much better now :+1:

**Comment by christyjacob4** [comment]
THANK YOU so much! All changes merged 🥳
Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swag!

**Pull request merged**